src/Reveal.Sdk.Dom.Tests/Visualizations/Extensions/IVisualizationExtensionsFixture.cs

```diff
@@ -0,0 +1,335 @@
+﻿using Moq;
+using Newtonsoft.Json;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Newtonsoft.Json;
+using System.Reflection;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations.Extensions
+{
+    public class IVisualizationExtensionsFixture
+    {
+        [Fact]
+        public void SetPosition_UpdateRowAndColumnSpan_WithoutCondition()
+        {
+            // Arrange
+            var mockVisualization = new Mock<IVisualization>();
+            mockVisualization.SetupProperty(v => v.RowSpan);
+            mockVisualization.SetupProperty(v => v.ColumnSpan);
+            var visualization = mockVisualization.Object;
+
+            var rowSpan = 10;
+            var columnSpan = 20;
+
+            // Act
+            var updatedVisualization = visualization.SetPosition(rowSpan, columnSpan);
+
+            // Assert
+            Assert.Equal(rowSpan, visualization.RowSpan);

```

Comment:
although `visualization` and `updatedVisualization` are the same, it should be `updatedVisualization.RowSpan` here in the assert. It's passing, but IMO it's not correct to check like that if you reassigned it to `updatedVisualization`